docs/userGuide.md

```diff
@@ -51,33 +51,17 @@ for being autoprovisioned when they register in the agent.
 devices to different statically configured types.
 
 # Testing
-The IoT Agent comes with a test suite to check the main functionalities. In order to execute the test suite you must have the Grunt client installed. You can install it using the following command (you will need root permissions):
+The IoT Agent comes with a test suite to check the main functionalities. Yoiu can execute the tests using:
```

Comment:
Typo - Yoiu => You

Reply:
Fixed with 2c1e34f10ba2a2eefa7a787b4ebf15ceb1cf88dc